vsts-api-patcher/src/main.rs

```diff
@@ -684,6 +685,26 @@ impl Patcher {
         }
     }
 
+    fn patch_probation_retries_type(
+        &mut self,
+        key: &[&str],
+        value: &JsonValue,
+    ) -> Option<JsonValue> {
+        // Only applies to hooks specs
+        if !self.spec_path.ends_with("serviceHooks.json") {
+            return None;
+        }
+        match key {
+            ["definitions", "Subscription", "properties", "probationRetries"] => {
+                let mut value = value.clone();
+                value["type"] = JsonValue::from("number");
+                value["format"] = JsonValue::from("float");
```

Comment:
I think that this field should be an integer rather than a float, given the data from the server and the name of the field.
I therefore think you should change this to...
```
                value["type"] = JsonValue::from("integer");
                value["format"] = JsonValue::from("int32");
```

Reply:
Thanks for fixing up!